app/kumactl/pkg/cmd/root_context.go

```diff
@@ -105,6 +111,21 @@ func (rc *RootContext) CurrentDataplaneOverviewClient() (kumactl_resources.Datap
 	return rc.Runtime.NewDataplaneOverviewClient(controlPlane.Coordinates.ApiServer)
 }
 
+func (rc *RootContext) CurrentDataplaneTokenClient() (tokens.DataplaneTokenClient, error) {
+	controlPlane, err := rc.CurrentControlPlane()
+	if err != nil {
+		return nil, err
+	}
+	// todo(jakubdyszkiewicz) this will be replaced by inferring addresses https://github.com/Kong/kuma/issues/315
+	apiServerUrl, err := url.Parse(controlPlane.Coordinates.ApiServer.Url)
+	if err != nil {
+		return nil, err
+	}
+	host, _, err := net.SplitHostPort(apiServerUrl.Host)
+	const defaultDpTokenPort = 5679
+	return rc.Runtime.NewDataplaneTokenClient(fmt.Sprintf("http://%s:%d", host, defaultDpTokenPort))
```

Comment:
Protocol can be `http` only on `loopback` address. Otherwise, always `https`

Reply:
It won't work without providing certs anyways. I'll fix that when adding support for public https port.